src/main/java/com/amazon/ion/impl/IonReaderTextSystemX.java

```diff
@@ -1211,6 +1211,8 @@ protected Macro loadMacro() {
             if (IonReaderTextSystemX.this.nextRaw() == null) {
                 throw new IonException("Macro invocation missing address.");
             }
+            List<SymbolToken> annotations = getAnnotations();
+            boolean isSystemMacro = !annotations.isEmpty() && SystemSymbols_1_1.ION.getText().equals(annotations.get(0).getText());
```

Comment:
The added `macroInvocationInTaggedExpressionGroup` test exercises this because it invokes the `values` macro from a context that does not export the Ion 1.1 system macros. The invocation looks like `(:$ion::values ...)`